src/main/java/com/xebialabs/overthere/gcp/credentials/ProjectCredentials.java

```diff
@@ -13,10 +14,20 @@
         this.clientEmail = clientEmail;
     }
 
+    ProjectCredentials(final Credential credential, final String projectId, final String clientEmail) {
+        this.credential = credential;
+        this.projectId = projectId;
+        this.clientEmail = clientEmail;
+    }
+
     public Credentials getCredentials() {
         return credentials;
     }
 
+    public Credential getCredential() {
```

Comment:
Can we improve the naming of this member, for example:

private Credential oauth2Credential;
...
public Credential getOauth2Credential() {
....

Reply:
Sure will change the same :)